lib/presentation/shared_widgets/crowdaction_card.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+
+import '../themes/constants.dart';
+
+class CrowdActionCard extends StatelessWidget {
+  final String title;
+  final String imagePath;
+  final List<Widget> chips;
+  final String? description;
+  const CrowdActionCard({
+    Key? key,
+    required this.title,
+    required this.imagePath,
+    required this.chips,
+    this.description,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: const EdgeInsets.all(12.0),
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(20.0),
+        boxShadow: const [
+          BoxShadow(
+            color: kShadowColor,
+            blurRadius: 4.0,
+            offset: Offset(0, 4),
+          ),
+        ],
+      ),
+      child: Container(
+        decoration: BoxDecoration(
+          borderRadius: BorderRadius.circular(20.0),
+          color: kSecondaryColor,
+        ),
+        child: Column(
+          children: [
+            Container(
+              height: 216,
+              decoration: BoxDecoration(
+                borderRadius: const BorderRadius.only(
+                  topLeft: Radius.circular(20.0),
+                  topRight: Radius.circular(20.0),
+                ),
+                image: DecorationImage(
+                  fit: BoxFit.fill,
+                  image: AssetImage(imagePath),
```

Comment:
Let's go with NetworkImage. This way we can fetch the image path from a CrowdAction model later and don't have to admit the image in our source files.

Reply:
Thanks! I changed it to `NetworkImage`.